crypto/cversion.c

```diff
@@ -76,6 +76,11 @@ const char *OpenSSL_version(int t)
 #else
         return "MODULESDIR: N/A";
 #endif
+    case OPENSSL_CPU_INFO:
+        {
+            const char *cpu_info = OPENSSL_info(OPENSSL_INFO_CPU_SETTINGS);
+            return cpu_info != NULL ? cpu_info : "CPUINFO: N/A";
+        }
```

Comment:
The troublesome part with this is that the output from this option doesn't match the rest when `OPENSSL_INFO_CPU_SETTINGS` has a value.  All other returned strings have a lable, but this one only has it when this option isn't applicable.  That makes `openssl version -a` look a bit weird...

I'm a bit ambivalent about that, but don't really care more than to make this remark and leave the choice to you.
(another idea is to refactor `openssl version` to use `OPENSSL_info` and handle all the labling itself, but that's for another PR, methinks)

Reply:
Yes, I will think about it, maybe I will need another RUN_ONCE to add the label.

Reply:
holy shit.  please suggest a name.

Reply:
What do you mean?  Or is that last comment misplaced?

Reply:
i ment in response to your other comment, but github is to blame :)